module/config.mjs

```diff
@@ -1237,6 +1237,7 @@ DND5E.itemProperties = {
   },
   mgc: {
     label: "DND5E.Item.Property.Magical",
+    icon: "systems/dnd5e/icons/svg/item-properties/magical.svg",
```

Comment:
Small preference to naming this new folder just `properties`.

Reply:
Also, we use the Font Awesome's wand icon to represent magic damage in the resistances section. We might want to standardize on this new icon there since we can't use font awesome for properties icons, but that should probably be a followup PR.

Reply:
I'll create an issue so we don't forget.

Reply:
Actually, I'm not seeing it being used there, but maybe I'm thinking of the wrong spot?

![Screenshot_20240206_214953](https://github.com/foundryvtt/dnd5e/assets/1226888/41c98756-588d-43c1-8ae2-fcb983b857d8)

Reply:
Its here on the main sheet:

<img width="136" alt="Screenshot 2024-02-06 at 20 53 19" src="https://github.com/foundryvtt/dnd5e/assets/19979839/d3e0d59e-e016-4438-b129-c278e147599f">